src/components/main-page/Hero.tsx

```diff
@@ -0,0 +1,101 @@
+import CellImage from "@/assets/cell.png";
+import Image from "next/image";
+import { AnimatePresence, motion } from "framer-motion";
+import { useRef, useState } from "react";
+import Link from "next/link";
+
+export default function Hero() {
+  return (
+    // Landing hero
+    <div>
+      <div
+        className="flex flex-row flex-wrap items-center justify-center
+                xl:flex-nowrap "
+      >
+        <Image
+          className="xl:w-[50rem] xl:h-[50rem]"
+          src={CellImage}
+          alt="Hand-drawn Image of Cell"
+        />
+        <div
+          className="flex flex-col w-fit items-center gap-y-[1.0rem]
+                    xl:items-start xl:gap-y-[2.25rem]"
+        >
+          <h1
+            className="text-primary text-[3.5rem] text-center
+                        md:text-[6rem] xl:text-left"
+          >
+            UBC BIOMOD
+          </h1>
+          <h2
+            className="text-center text-subHeadingRegMob
+                        md:text-subHeadingReg xl:text-left"
+          >
+            This is gonna say something so super duper cool about biomod that
+            everyone will freak and scroll through the entire website!
+          </h2>
+          <Link
+            href={"/about"}
+            className="bg-accent grow-0 px-[2.5rem] py-[1rem] rounded-[0.75rem]
```

Comment:
Factor button style out into a component or class, since we're gonna be reusing it everywhere :3

Reply:
probably later when we clean up the website.